### PR TITLE
Prevent plugin django stack loading in plugins

### PR DESCRIPTION
**Pull request by benjaoming**
### Summary

This was started a while back, opening a PR to remind us that this should happen if we don't want module load order complexity to get out of hand.

### Reviewer guidance

Should find a priority for this.

### References

#2510

----

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [x] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by rtibbles** [comment]
This looks important to me (and will be helpful for the build decoupling work also). Could we update this PR to manage the merge conflict?

**Comment by benjaoming** [comment]
This should not impact the Perseus plugin, as it doesn't load any `urls` module in `kolibri_module`

**Comment by benjaoming** [comment]
@rtibbles provided that tests pass, this should be ready now

**Comment by rtibbles** [comment]
Tests still failing :/

File: kolibri/plugins/style_guide/kolibri_plugin.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 from kolibri.core.webpack import hooks as webpack_hooks
 from kolibri.plugins.base import KolibriPluginBase
-from . import hooks, urls
+
+from . import hooks
 
 
 class StyleGuide(KolibriPluginBase):
     def url_module(self):
+        from . import urls
```
Comment (rtibbles) [inline kolibri/plugins/style_guide/kolibri_plugin.py:11]: Might it be worth doing this by convention?

**Comment by rtibbles** [comment]
I don't know why what you are doing here has caused this test failure to arise. Seems to be on the Python 2.7 build, where the specific behaviour for Python 2 is not working?
`exc_message = getattr(e, 'message', getattr(e, 'msg'))` - it is not getting a message from the first attempt at getattr.

**Comment by rtibbles** [comment]
This is great, and will allow us to do the frontend building without invoking Kolibri and the full Django stack!

**Pull request merged**